community/socklog/socklog.setup

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+NORMAL="\033[1;0m"
+STRONG="\033[1;1m"
+RED="\033[1;31m"
+GREEN="\033[1;32m"
+runscript=/etc/sv/socklog-unix/run
+
+print_stop() {
+	local prompt="${STRONG}$1 ${RED}$2${NORMAL}"
+	printf "${prompt} %s"
```

Comment:
This doesn’t look right. The first argument is a format, you’ve used format character `%s`, but didn’t provide an argument for it.